daisy/step_wait_for_instances_signal.go

```diff
@@ -224,9 +224,8 @@ func (w *WaitForInstancesSignal) run(ctx context.Context, s *Step) DError {
 			}
 			if is.SerialOutput != nil {
 				go func() {
-					if err := waitForSerialOutput(s, m["project"], m["zone"], m["instance"], is.SerialOutput, is.interval); err != nil {
-						e <- err
-					}
+					// send a signal to end other waiting serial ports, no matter whether an error happens
+					e <- waitForSerialOutput(s, m["project"], m["zone"], m["instance"], is.SerialOutput, is.interval)
```

Comment:
We don't want to do this as a single WaitForInstancesSignal can wait for multiple instances and we do want to actually wait for all of them

Reply:
In my use-case, 'failure' is thrown from port 3 while 'success' is caught from port 1. if "WaitForInstancesSignal" is designed for "all" logic, we need another "WaitForAnyInstancesSignal". Or, we need to ignore waitings for 'failure' when we caught a 'success'. Which one do you think is better? I prefer a new 'WaitForAnyInstancesSignal'

Reply:
Or, to be able to wait on multiple ports for a single "Wait"

Reply:
I added "WaitForAnyInstancesSignal", please have a look